src/selectors/DynamicContext.js

```diff
@@ -52,10 +52,42 @@ class DynamicContext {
 	createScopedContext (overlayContext) {
```

Comment:
Remove entirely / replace with scopeWithEverything (without the !== undefined ternaries)